src/App.tsx

```diff
@@ -10,15 +14,18 @@ import { parseCSV, Payment } from "./parser";
 import { MessageContext } from "./contexts/MessageContextProvider";
 
 const App: React.FC = () => {
-  const { sdk, safe } = useSafeAppsSDK();
+  const { sdk } = useSafeAppsSDK();
   const { tokenList, isLoading } = useTokenList();
   const [submitting, setSubmitting] = useState(false);
   const [transferContent, setTransferContent] = useState<Payment[]>([]);
   const [csvText, setCsvText] = useState<string>(
     "token_address,receiver,amount,decimals"
   );
-
   const { addMessage, setMessages } = useContext(MessageContext);
+  // const web3Provider = useMemo(
+  //   () => new ethers.providers.Web3Provider(new SafeAppProvider(safe, sdk)),
+  //   [sdk, safe]
+  // );
```

Comment:
Gotta leave this stuff here, because otherwise I will forget. It took a bunch of digging to find this.